index.rst

```diff
@@ -15,5 +10,265 @@ Change Record
+  Level 2 milestone
+
+* DM's TCT approval for Level 3 milestones
+
+* no approval is needed for Level 4 milestones, however any non-trivial change need to be communicated
+  to the DM team
+
+* trivial changes, such as small corrections / clarifications to narrative that do not affect
+  scope, time or budget are allowed without approval.
+
+Typically, adjustments to Level 1, 2 and 3 milestones are made every 6 or 12 months. Changes to Level 4
+milestones can occur more frequently.
+
+Short-term Planning
+-------------------
+
+Short-term Plan is managed through *epics* and *stories*, and executed through *cycles*.
+
+In the EVMS world, epics map directly to *activities*, and stories map to *activity steps*.
+
+Epics and Stories
+~~~~~~~~~~~~~~~~~
+
+Typically, planning packages encompass relatively large blocks of work. To plan and execute work
+in details, finer-grain planning is required. This is implemented through epics and stories.
+
+Each epic captures a non-trivial work that is associated with a subset of work defined in a
+planning package. Each epic has a clear description, well defined deliverables, and
+relationships with a planning package (an epic *implements* a PP), and a Milestone (an epic
+*blocks* a milestone). Epics are resources loaded, and have start and end dates defined.
+
+Each epic is broken into smaller chunks of work, called *Stories*. Stories are primarily used
+to define and manage short-term activities of individual developers.
+
+Epics and stories are used to track all software work, as well as work on delivering hardware and
+services that are tightly coupled with software (example: "Deliver OpenStack-based Test Cluster
+running RedHat 6.1".) Details of work on standalone activities do not have to be captured on daily
+bases through stories (example: "Deliver 10Gbit link between Chile and USA")
+
+.. _cycles-and-releases:
+
+Cycles and Releases
+~~~~~~~~~~~~~~~~~~~
+
+The work is executed through 6-month *cycles*. Scope of work for each cycle is driven by the Baseline
+Plan, in particular by milestones. Each cycle ends with a new software release. Releases are
+thoroughly tested and documented.
+
+Cycles are planned through *epics*. Each epic must be defined such that it can be completed
+within a single cycle.
+
+Once a plan is defined for a given cycle, it is loaded into PMCS and changed-controlled. A plan for
+a cycle is loaded to PMCS during the month proceeding the start of the cycle.
+
+Any non-trivial adjustments to the plan that affect scope, schedule or budget must be approved
+by CCB.
+
+It is acceptable to load the plan in 3-month chunks, e.g., the plan for
+the first 3 months of the cycle is loaded before the start of the cycle, and the remaining
+part of the plan covering the last 3 months is loaded before the 4th month starts. This
+allows for minor fine-tuning of the second half of the cycle without going through the CCB
+approval.
+
+Epics-based Long-term Planning
```

Comment:
I guess we (I) never made a comprehensive pass through the document thinking whether any info should be pushed out of this document and moved into the tech note. Take it, I'll remove it from here (or even better, remove it in your branch and I'll merge your changes in)